library/std/src/ffi/os_str.rs

```diff
@@ -1222,6 +1222,23 @@ impl OsStr {
     }
 }
 
+#[unstable(feature = "slice_concat_ext", issue = "27747")]
```

Comment:
The tracking issue #27747 was closed a while ago.

Reply:
I used the same tracking issue the other impl blocks for that trait have. The reason to close the tracking issue was because the feature is to be perma-unstable: https://github.com/rust-lang/rust/issues/27747#issuecomment-925420642 . I'm not sure about changes in policy, but in the older times tracking issues for perma-unstable things would remain open indefinitely.